susemanager/src/mgr_bootstrap_data.py

```diff
@@ -159,50 +159,6 @@
         "venv-salt-minion",
 ]
 
-PKGLISTUMBLEWEED_SALT_NO_BUNDLE = [
```

Comment:
We don't what the non-bundle package anymore. With the bundle we don't depend on the version Tumbleweed has, so we can always stay compatible with the Salt Master on the Uyuni Server.